solnlib/log.py

```diff
@@ -306,9 +332,22 @@ def log_exception(
     full_msg: bool = True,
     msg_before: str = None,
     msg_after: str = None,
+    exc_label: str = None,
     log_level: int = logging.ERROR,
 ):
     """General function to log exceptions."""
+    msg = _get_exception_message(e, full_msg, msg_before, msg_after)
+    if exc_label:
+        msg = f'exc_l="{exc_label}" {msg}'
```

Comment:
@artemrys what if we add "Other" as a default value for `exc_lable`? This way every exception logged with `log_exception` function without `exc_label` will be classified as "Other"

Reply:
this argument is now mandatory, so must always be defined by the user (breaking change)